iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/anonymizer/ConverterContext.java

```diff
@@ -4,16 +4,16 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.iidm.xml;
+package com.powsybl.iidm.anonymizer;
```

Comment:
Move this class to com.powsybl.iidm package: one day, we'll probably want to use the ConverterContext to handle something else that only an anonymizer instance